terasoluna-gfw-functionaltest-domain/src/main/java/org/terasoluna/gfw/functionaltest/domain/service/codelist/CodeListServiceImpl.java

```diff
@@ -46,7 +46,7 @@
 
     @Override
     public ItemCode findOne(Integer code) {
-        return codeListRepository.findOne(code);
+        return codeListRepository.findById(code).orElse(null);
```

Comment:
- [ ] Also, you may have to rethink the name of the service `findOne` method. (new guideline issue?)